test/service/test_repository_projects.py

```diff
@@ -104,3 +111,46 @@ def assert_file_type_summaries(hit):
             self.assertEqual(hit, single_hit)
         self.assertIn('pagination', response_json)
         self.assertIn('termFacets', response_json)
+
+
+@patch_dss_endpoint
+@patch_source_cache
+class TestProjectPublications(WebServiceTestCase):
```

Comment:
We can't afford to add a full test plus canned bundle for every property we add.